container/openejb-jee/src/test/resources/web-fragment-example.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file misses a license header - subsequently, the license header checks in CI will fail. 

You can ake a look at the other descriptor files and just copy & paste the ASF license header into this file.

Reply:
I pasted the license header, but I have not commited yet.